### PR TITLE
add experimental Windows support to home page

### PR DESCRIPTION
**Pull request by kelseiv**
Include Windows is experimental on home page per https://github.com/influxdata/docs.influxdata.com/issues/2621

### CODE REVIEW

File: layouts/index.html [hunk 1]
```diff
@@ -6,7 +6,7 @@
 	<div class="homepage--card card-platform">
 		<div class="homepage--card-container">
 			<h1 class="homepage--card-title">InfluxData Platform</h1>
-			<p class="homepage--card-description">The leading time series platform built for metrics and events.</p>
+			<p class="homepage--card-description">The leading time series platform built for metrics and events. Fully supported on multiple operating systems, including Linux, MacOS, and Unix. Windows not fully supported and provided for experimental use only.</p>
```
Comment (sanderson) [inline layouts/index.html]: - MacOS -> macOS (this is the trademarked form of the name)
- Windows support is experimental.

Comment (kelseiv) [reply layouts/index.html]: Thanks, @sanderson. Will add to install docs for sure. Wanted to get input on text first. While the request was originally to include on doc homepage and/or download page itself, I'm open to removing it from docs home page. It does get the fact Windows is experimental out in front.

Comment (kelseiv) [reply layouts/index.html]: Thanks, @sanderson. Will add to install docs for sure. Wanted to get input on text first. While the request was originally to include on doc homepage and/or download page itself, I'm open to removing it from docs home page. It does get the fact Windows is experimental out in front.

Comment (kelseiv) [reply layouts/index.html]: Thanks, @sanderson. Will add to install docs for sure. Wanted to get input on text first. While the request was originally to include on doc home page and/or download page itself, I'm open to removing it from docs home page. It does get the fact Windows is experimental out in front.

Comment (sanderson) [reply layouts/index.html]: It does, but I don't know that it's apparent enough. It get's buried in a wall of images/text and I don't know how many people will actually read it.

Comment (kelseiv) [reply layouts/index.html]: @sanderson, very true. Think we all agree on install docs note.

@sjwang90, @sanderson, @noramullen1, @pierwill ...thoughts on possible other options?:
-  leave info off our main docs page
- note on main docs page under the Start Here button 
- note on download portal
- mention in install docs is all we need

Comment (kelseiv) [reply layouts/index.html]: Regarding the text  "Windows support is experimental" 

Could this be interpreted:
- We don't support Windows, but you can experiment 
OR
- We’re striving to support Windows but still in experimental phase/testing 

Perhaps we're not sure yet? Want to set expectations on what to expect for support.

Comment (sanderson) [reply layouts/index.html]: I think both of those interpretations are true.

Comment (sjwang90) [reply layouts/index.html]: I think these two would be the best options:
- note on download portal (only Under Windows downloads links)
- mention in install docs is all we need

Don't need it on any main pages, wouldn't want to detract any users.  

I agree on both the interpretations.

Comment (kelseiv) [reply layouts/index.html]: Thanks, @sanderson @sjwang90, updated all install docs and slacked @ChrisChurilo to see if she can help us add text "Windows support is experimental"
to download portal (only Under Windows downloads links)

Comment (kelseiv) [reply layouts/index.html]: per @lukevmorris, download portal issues can be submitted to [influxdb-customers repo](https://github.com/influxdata/influxdb-customers). Thank you, @sjwang90 for submitting another issue on that repo--closing this one.

**Pull request merged**